### PR TITLE
Docs(/techniques/compression):  Brotli compression performance

### PR DESCRIPTION
**Pull request by eugleenyc**
This adds notes on  Brotli compression options and provides an example.

Closes issue #2834

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Docs
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #2834

## What is the new behavior?
Added documentation regarding Brotli performance.

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: content/techniques/compression.md [hunk 1]
```diff
@@ -38,7 +38,15 @@ import compression from '@fastify/compress';
 await app.register(compression);
 ```
 
-By default, `@fastify/compress` will use Brotli compression (on Node >= 11.7.0) when browsers indicate support for the encoding. While Brotli is quite efficient in terms of compression ratio, it's also quite slow. Due to this, you may want to tell fastify-compress to only use deflate and gzip to compress responses; you'll end up with larger responses but they'll be delivered much more quickly.
+By default, `@fastify/compress` will use Brotli compression (on Node >= 11.7.0) when browsers indicate support for the encoding. While Brotli can be quite efficient in terms of compression ratio, it can also be quite slow. By default, Brotli sets a maximum compression quality of 11, although it can be adjusted to reduce compression time in lieu of compression quality by adjusting the `BROTLI_PARAM_QUALITY` between 0 min and 11 max. This will require fine tuning to optimize space/time performance. An example with quality 4: 
+
+```Typescript
```
Comment (Tony133) [inline content/techniques/compression.md:43]:
```suggestion
```typescript
```

File: content/techniques/compression.md [hunk 2]
```diff
@@ -38,7 +38,15 @@ import compression from '@fastify/compress';
...
+import { constants } from 'zlib';
+// somewhere in your initialization file
+await app.register(compression, { brotliOptions: { params: { [constants.BROTLI_PARAM_QUALITY]: 4 } } });
+```
+
+To simplify, you may want to tell fastify-compress to only use deflate and gzip to compress responses; you'll end up with potentially larger responses but they'll be delivered much more quickly.
```
Comment (Tony133) [inline content/techniques/compression.md:49]:
```suggestion
To simplify, you may want to tell `fastify-compress` to only use deflate and gzip to compress responses; you'll end up with potentially larger responses but they'll be delivered much more quickly.
```

**Review by Tony133 — CHANGES_REQUESTED** [review]

**Comment by Tony133** [comment]
I pointed out two small changes, otherwise it seems as good as written. 😉👍

**Comment by eugleenyc** [comment]
@Tony133 added!

**Review by Tony133 — APPROVED** [review]

**Pull request merged**